src/main/webapp/app/exercises/programming/manage/code-editor/code-editor-instructor-base-container.component.ts

```diff
@@ -109,7 +111,7 @@ export abstract class CodeEditorInstructorBaseContainerComponent extends CodeEdi
                     catchError(() => throwError('exerciseNotFound')),
                     tap((exercise) => {
                         this.exercise = exercise;
-                        this.isExamMode = !exercise.course;
+                        this.course = exercise.course ? exercise.course : exercise.exerciseGroup!.exam!.course;
```

Comment:
How about the shortened version?
```suggestion
                        this.course = exercise.course ?? exercise.exerciseGroup!.exam!.course;
```

Reply:
Cool didn't know this feature